### ISSUE TITLE
Why not targets [1:-1]? 

### ISSUE THREAD

**Issue by openworld-lab** [NExT-GPT/NExT-GPT#67]
**code/model/anyToImageVideoAudio.py line 386-387**

`chosen_tokens = torch.max(outputs.logits, dim=-1)[1][:, 1:-1]  # [B, S-1]
labels = targets[:, 2:]`

**Comment by vadeli** [comment]
@BinZhu-ece, Same question. Did you figure it out?
